pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -961,21 +1094,42 @@ func afwSchedulerEntry(client splcommon.ControllerClient, cr splcommon.MetaObjec
 	}
 	scopedLog.Info("List of cluster scoped apps(appName:digest) for this reconcile entry", "apps", clusterAppsList)
 
+	targetPodName := getApplicablePodNameForWorker(cr, 0)
```

Comment:
nit: Now that it has other use, probably, we can change the name to something like `getApplicablePodNameForAppFramwork()`?